README.md

```diff
@@ -20,10 +20,18 @@
 
 <img src="https://user-images.githubusercontent.com/1153921/95269366-1b814680-07ee-11eb-8ff4-c1b0b8758499.png" alt="---" style="max-width: 100%;" />
 
-Netdata's **distributed, real-time monitoring Agent** collects thousands of metrics from systems, hardware, containers,
```

Comment:
kinda like the "thousands of metrics" phrase as is a bit of a differentiator tbh then just basic standard metrics from some other agents.